manifests/charts/istiod-remote/templates/configmap-jwks.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.pilot.jwksResolverExtraRootCA }}
```

Comment:
would this be configurable from istiod-remote?

Reply:
I guess so since you copied this to Istiod-remote's values.yaml.

Reply:
removed this